style.css

```diff
@@ -406,3 +536,305 @@ footer > hr {
+    font-size: 54px;
+    width: 41%;
+  }
+
+  p.main-text {
+    width: 60%;
+  }
+
+  .main-container {
+    background-color: white;
+  }
+
+  .works-1 {
+    background: #fff;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 2;
+    border-radius: 0;
+    row-gap: 0;
+    column-gap: 12px;
+    margin-top: 4px;
+    margin-bottom: 24px;
+    margin-left: 30px;
+    margin-right: 0;
+  }
+
+  .works {
+    background: white;
+    margin-top: 14px;
+  }
+
+  .works-head {
+    display: flex;
+    column-gap: 8px;
+    margin-left: 28px;
+    margin-bottom: 34px;
+  }
+
+  .section-header {
+    width: 315px;
+    height: 52px;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+  }
+
+  .section-pargraph {
+    width: 566px;
+    height: 148px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+  }
+
+  .line-horizontal {
+    width: 794px;
+    height: 0;
+    border: 1px solid #212121;
+  }
+
+  .img-wrapper {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    column-gap: 44px;
+    padding-left: 34px;
+    background: white;
+  }
+
+  .mult-story {
+    width: 60%;
+    height: 400px;
+  }
+
+  .project-title {
+    width: 448px;
+    height: 48px;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    margin-bottom: 6px;
+    display: flex;
+    align-items: center;
+    color: #091e42;
+  }
+
+  .project-description {
+    width: 448px;
+    height: 140px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #091e42;
+  }
+
+  .project-data > .project-title {
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 44px;
+    color: #fff;
+    padding-bottom: 24px;
+  }
+
+  .project-data > .project-description {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 17px;
+    line-height: 24px;
+    color: #fff;
+  }
+
+  .fullwidth-button {
+    display: block;
+    width: 369px;
+  }
+
+  .fullwidth-button-1 {
+    display: none;
+    width: 369px;
+    height: 435px;
+  }
+
+  .languages-1 {
+    display: none;
+  }
+
+  .about-me {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #1c1a19;
+    background: #262626;
+  }
+
+  .about-me-main {
+    margin-left: 80px;
+  }
+
+  .about-me > .about-me-main-1 {
+    background-position: center;
+    background-repeat: no-repeat;
+    height: 652px;
+    width: 652px;
+  }
+
```

Comment:
please try to align your About myself heading to the left following the design from figma, your design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/200122334-ffca016b-5161-47d5-b922-9fc5f7470f7c.png)
 figma design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/200122381-fc786067-f666-4d9d-b7bb-e2a11e0d89a1.png)